src/en/getting-started-jaas.md

```diff
@@ -244,8 +244,8 @@ you can manually copy and paste the unique authorisation URL from the command
 output.
 
 After successful authentication, you will be asked to enter a descriptive name
-for the JAAS controller. This means any models or applications you have
-already deployed are now accessible from the command line.
+for the JAAS controller. This means you can operate your models or
+applications in JAAS from the command line.
```

Comment:
No apology needed. I didn't look at it right from your last comment. I've updated it and agree.